examples/experimental/dagster-airlift/dagster_airlift_tests/unit_tests/conftest.py

```diff
@@ -41,7 +44,7 @@ def fully_loaded_repo_from_airflow_asset_graph(
     assets_per_task: Dict[str, Dict[str, List[Tuple[str, List[str]]]]],
     additional_defs: Definitions = Definitions(),
     create_runs: bool = True,
-    event_transformer_fn: Optional[DagsterEventTransformerFn] = None,
+    event_transformer_fn: DagsterEventTransformerFn = default_event_transformer,
```

Comment:
Not a huge deal as it is in test code, but I definitely find it to be a "smell' when defaults are specified multiple times like this. The decision to decide what default it should be done once IMO, with the rest of them defaulting to None.

Reply:
I agree in general but in this case since there are multiple different layers of usable API surface area in these tests, I think it's necessary.